extensions/cornerstone/src/initMeasurementService.js

```diff
@@ -66,14 +68,25 @@ const initMeasurementService = (
     ArrowAnnotate.toMeasurement
   );
 
+  // On the UI side, the Calibration Line tool will work almost the same as the
+  // Length tool
+  MeasurementService.addMapping(
+    csTools3DVer1MeasurementSource,
+    'CalibrationLine',
+    Length.matchingCriteria,
+    Length.toAnnotation,
+    Length.toMeasurement
+  );
```

Comment:
I don't understand why you are adding it to the measurementService? The tools added here store state in the measurementService, and need mappings. But to me, the calibration tool is a tool that runs a call back on finish. Do we really need mapping here?

Reply:
You're right, the code is not necessary